<launch>
  <arg name="map_file" default="$(find utexas_gdc)/maps/3ne.yaml" />
  <arg name="door_file" default="$(find clingo_helpers)/config/doors.yaml" />
  <arg name="location_file" default="$(find clingo_helpers)/config/locations.yaml" />
  <node name="clingo_interface" type="clingo_interface" pkg="clingo_interface">
    <rosparam file="$(find clingo_helpers)/config/costmap.yaml" command="load" />
    <param name="map_file" value="$(arg map_file)" />
    <param name="door_file" value="$(arg door_file)" />
    <param name="location_file" value="$(arg location_file)" />
  </node>
</launch>
